qt/scientific_interfaces/test/ISISReflectometry/RunsTable/RunsTablePresenterTest.h

```diff
@@ -116,6 +116,18 @@ class RunsTablePresenterTest {
                              Cell(""),     Cell(""),    Cell("")};
   }
 
+  std::vector<Cell> cellsArrayWithQValues(std::string const &run = "12345",
```

Comment:
This doesn't seem to be used?

Reply:
No it doesn't appear to. Can't remember what my thought track there was so unless you can think of a use case I'll just remove it.